apis/v1alpha2/validation/httproute_test.go

```diff
@@ -312,12 +314,10 @@ func TestValidateHTTPRoute(t *testing.T) {
 	}
 	for _, tt := range tests {
 		// copy variable to avoid scope problems with ranges
-		tt := tt
-		t.Run(tt.name, func(t *testing.T) {
```

Comment:
This should still be wrapped in `t.Run`

Reply:
9be59b9deca99e1af9fbdab5fc6a20a90bacd7dc